io/eolearn/io/geometry_io.py

```diff
@@ -40,6 +41,10 @@ def __init__(self, feature, reproject=True, clip=False, config=None):
         self.reproject = reproject
         self.clip = clip
 
+    @abc.abstractmethod
```

Comment:
If you want this to be a true abstract base class don't forget to set the meta class